### PR TITLE
Import bootst 2improve

### PR DESCRIPTION
**Pull request by bradymiller**
testing well and plan to bring this in when pass travis-ci

### CODE REVIEW

File: custom/import_xml.php [hunk 1]
```diff
@@ -190,24 +193,31 @@ function setInsurance($pid, $ainsurance, $asubscriber, $seq) {
...
-
-<p>
-<input type='submit' name='form_import' value=<?php xl('Import Patient','e','\'','\''); ?> /> &nbsp;
-<input type='button' value=<?php xl('Cancel','e','\'','\''); ?> onclick='window.close()' /></p>
+<form method='post' action="import_xml.php" onsubmit="return top.restoreSession()">
+    <div class="container">
+        <div class="row">
+            <div class="col-xs-12">
+                <div class="form-group"></div>
+                <div class="form-group">
```
Comment (robertdown) [inline custom/import_xml.php]: Empty div?

File: custom/import_xml.php [hunk 2]
```diff
@@ -190,24 +193,31 @@ function setInsurance($pid, $ainsurance, $asubscriber, $seq) {
...
+                    <textarea name='form_import_data' class='form-control' rows='10' cols='50'></textarea>
```
Comment (robertdown) [inline custom/import_xml.php]: Textareas inside of bootstrap don't need `cols` attribute, just `rows`

File: custom/import_xml.php [hunk 3]
```diff
@@ -190,24 +193,31 @@ function setInsurance($pid, $ainsurance, $asubscriber, $seq) {
...
+                </div>
+                <div class="form-group text-right">
+                    <div class="btn-group" role="group">
+                        <button type='submit' class='btn btn-default btn-save' name='form_import' value='bn_import'>
+                            <?php echo xla('Import'); ?>
```
Comment (robertdown) [inline custom/import_xml.php]: `xla`?

File: custom/import_xml.php (hunk 1 above)
Comment (bradymiller) [inline custom/import_xml.php]: if don't do this, then the below textarea hugs right at the top of the form and it looks really weird. LEt me know if there's a better way to bring in spacing and I'll change this (in both the export and import scripts).

File: custom/import_xml.php (hunk 2 above)
Comment (bradymiller) [inline custom/import_xml.php]: will fix this in the import and export scripts

File: custom/import_xml.php (hunk 3 above)
Comment (bradymiller) [inline custom/import_xml.php]: I'll check, but I think I caught and fixed this in a subsequent commit.
(btw, that's a good catch and means you are well on your way to being a anal reviewer extraordinaire :))

File: custom/import_xml.php (hunk 2 above)
Comment (bradymiller) [inline custom/import_xml.php]: just committed the fix for this

File: custom/import_xml.php (hunk 3 above)
Comment (bradymiller) [inline custom/import_xml.php]: just committed the fix for this

**Pull request merged**